todoApp/src/App.tsx

```diff
@@ -23,29 +24,41 @@ const App = () => {
     setTodoTasks(todoArray);
   };
 
-  const curTime = new Date();
-  todoTasks = todoTasks.filter((todo: TodoListInterface) => {
-    let curDate = `${curTime.getDate()}/${
-      curTime.getMonth() + 1
-    }/${curTime.getFullYear()}`;
-    return curDate === todo.date;
-  });
-  localStorage.setItem("todo", JSON.stringify(todoTasks));
+  let curDate = `${moment().format("D")}/${moment().format(
```

Comment:
Move current date variable inside side effect and remove it from dependencies.
change it from `let` to `const`.